README.md

```diff
@@ -161,13 +155,33 @@ $ docker-compose run frontend rake update_pending_census_codes
 $ docker-compose run frontend rake update_providers_statistics
 ```
 
+## Updating boundaries
+
+When boundaries are updated each developer must reload their boundary tables:
+```bash
+$ docker-compose exec -T mysql mysql -u suyc -psuyc suyc <<< "TRUNCATE census_boundaries;"
+$ docker-compose exec -T mysql mysql -u suyc -psuyc suyc <<< "TRUNCATE zip_boundaries;"
+$ docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/zip_codes.sql
+$ docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/census_tracts.sql
+```
+
+>For Windows OS please use the following:
+```bash
+$ docker-compose exec -T mysql mysql -u suyc -psuyc suyc
+$ docker-compose exec -T mysql> TRUNCATE census_boundaries;
```

Comment:
So close, L170 starts a MySQL client, giving you MySQL prompt. you can drop `$ docker-compose exec -T ` from this line and the next two.

Reply:
$ docker-compose exec -T mysql> -u suyc -psuyc suyc
$ docker-compose exec -T mysql> TRUNCATE census_boundaries;
$ docker-compose exec -T mysql> TRUNCATE zip_boundaries;
$ docker-compose exec -T mysql> exit

Reply:
C:\Users\subli\SpeedUpAmerica>git status
On branch adding_details_to_setup
Your branch is up to date with 'origin/adding_details_to_setup'.

Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        new file:   -u
        modified:   README.md
        new file:   TRUNCATE

Reply:
C:\Users\subli\SpeedUpAmerica>docker-compose exec -T mysql> -u suyc -psuyc suyc

C:\Users\subli\SpeedUpAmerica>docker-compose exec -T mysql> TRUNCATE census_boundaries;

C:\Users\subli\SpeedUpAmerica>docker-compose exec -T mysql> TRUNCATE zip_boundaries;

C:\Users\subli\SpeedUpAmerica>git add .

C:\Users\subli\SpeedUpAmerica>git status
On branch adding_details_to_setup
Your branch is up to date with 'origin/adding_details_to_setup'.

Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        new file:   -u
        modified:   README.md
        new file:   TRUNCATE


C:\Users\subli\SpeedUpAmerica>git commit -m "added docker line"
[adding_details_to_setup 5ffd88f] added docker line
 3 files changed, 4 insertions(+), 2 deletions(-)
 create mode 100644 -u
 create mode 100644 TRUNCATE

C:\Users\subli\SpeedUpAmerica>git push
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 16 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 564 bytes | 564.00 KiB/s, done.
Total 5 (delta 3), reused 0 (delta 0)
remote: Resolving deltas: 100% (3/3), completed with 2 local objects.
To https://github.com/Hack4Eugene/SpeedUpAmerica.git
   9ab6075..5ffd88f  adding_details_to_setup -> adding_details_to_setup

C:\Users\subli\SpeedUpAmerica>

Reply:
What I get when I run those commands:
```
C:\Users\ryanr\repos\SpeedUpAmerica>docker-compose exec -T mysql> -u suyc -psuyc suyc

C:\Users\ryanr\repos\SpeedUpAmerica>docker-compose exec -T mysql> TRUNCATE census_boundaries;

C:\Users\ryanr\repos\SpeedUpAmerica>git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        -u
        TRUNCATE

nothing added to commit but untracked files present (use "git add" to track)
```

The issue is the first line is supposed to get you into a MySQL client and then you can run the truncate commands. What it's doing instead is making a -u and TRUNCATE file. If you look at the changes in this PR you can see that you added and committed those files, which need to be removed from the PR. Originally I provided you instructions like, but I incorrectly included a `-T`, causing it to not work as needed. I've doubled checked the instructions below, please try them too. 

The instructions should be:
```
> docker-compose exec mysql mysql -u suyc -psuyc suyc
mysql> TRUNCATE census_boundaries;
mysql>TRUNCATE zip_boundaries;
mysql> exit
> docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/zip_codes.sql
> docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/census_tracts.sql
```

When I run those instructions I get:
```
C:\Users\ryanr\repos\SpeedUpAmerica>docker-compose exec mysql mysql -u suyc -psuyc suyc
mysql: [Warning] Using a password on the command line interface can be insecure.
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 5
Server version: 5.7.26 MySQL Community Server (GPL)

Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql> TRUNCATE census_boundaries;
Query OK, 0 rows affected (0.02 sec)

mysql> TRUNCATE zip_boundaries;
Query OK, 0 rows affected (0.02 sec)

mysql> exit
Bye

C:\Users\ryanr\repos\SpeedUpAmerica>docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/zip_codes.sql
mysql: [Warning] Using a password on the command line interface can be insecure.

C:\Users\ryanr\repos\SpeedUpAmerica>docker-compose exec -T mysql mysql -u suyc -psuyc suyc < data/census_tracts.sql
mysql: [Warning] Using a password on the command line interface can be insecure.

C:\Users\ryanr\repos\SpeedUpAmerica>
```

Reply:
Thank you for the update. 

I have taken the updates from the notes and applied them. 

This was my experience:

`C:\Users\subli\SpeedUpAmerica>docker-compose exec mysql mysql -u suyc -psuyc suyc
mysql: [Warning] Using a password on the command line interface can be insecure.
Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 7
Server version: 5.7.26 MySQL Community Server (GPL)

Copyright (c) 2000, 2019, Oracle and/or its affiliates. All rights reserved.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql> TRUNCATE census_boundaries;
ERROR 1146 (42S02): Table 'suyc.census_boundaries' doesn't exist
mysql> exit
Bye

C:\Users\subli\SpeedUpAmerica>`

I'm sure you're familiar with this error as it seem to be a pretty common one. The 1146 error is corrupted data due to a number or reasons. 

Here is a website if you want to look further into this:

https://bobcares.com/blog/1146-table-doesnt-exist/

Long story short and not checking the website above;

There are a few ways that we can address this issue. 

1:) Restart MySQL server (aiding corrupted data)
2:) Repair the tables command: `myisamchk`
3:) Backup restore
4:) Copy ibdata file – If the ‘ibdata’ file is missing, we copy it from the backup and restore it to the data directory for MySQL, after discarding the tablespace to avoid any corruptions or errors.
5:) InnoDB crash recovery – In case where the backup is incomplete or ibdata file is also corrupt, we’ve still been able to recover the tables via our expert crash recovery methods.

Click this link to read more about the database crash rescue:

https://bobcares.com/blog/how-we-re-built-an-innodb-mysql-database-when-ibdata1-file-was-corrupted/

This is from a blog that was published back in 2017, so some things may of changed, but this was the best information I could find myself. It seems simple enough but I wanted to talk to you before fixing corrupted data.